### PR TITLE
refactor: use package version

### PR DESCRIPTION
**Pull request by andreivladbrg**
in light of [my comment](https://github.com/sablier-labs/docs/pull/186#discussion_r1867721145) and the package tethering change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,50 +28,50 @@ This repo aggregates all audits performed on Sablier smart contracts by third-pa
...
-| Cantina           | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-12-21](./lockup/v1.1/20231221_cantina.pdf) |
-| Cantina           | Firm  | [`07014ac`][07014ac], [`cc9434f`][cc9434f] | [2023-12-15](./lockup/v1.1/20231215_cantina.pdf) |
-| Turing Consulting | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-11-30](./lockup/v1.1/20231130_turing.pdf)  |
-| Cantina           | Firm  | [`79d88ca`][79d88ca]                       | [2023-09-06](./lockup/v1.1/20230906_cantina.pdf) |
+| Auditors          | Types | Commits                                    | Reports                                            |
+| :---------------- | :---- | :----------------------------------------- | :------------------------------------------------- |
+| Cantina           | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-12-21](./lockup/v1.1.0/20231221_cantina.pdf) |
+| Cantina           | Firm  | [`07014ac`][07014ac], [`cc9434f`][cc9434f] | [2023-12-15](./lockup/v1.1.0/20231215_cantina.pdf) |
+| Turing Consulting | Firm  | [`e5a227f`][e5a227f], [`0004fd2`][0004fd2] | [2023-11-30](./lockup/v1.1.0/20231130_turing.pdf)  |
```
Comment (smol-ninja) [inline README.md:50]: Calling it `v1.1` is better because some of them refer to `v1.1.0` while some of them refer to `v1.1.2`.

File: README.md [hunk 2]
```diff
@@ -28,50 +28,50 @@ This repo aggregates all audits performed on Sablier smart contracts by third-pa
...
+### Legacy
 
 [fc54b02]: https://github.com/sablier-labs/v1-protocol/tree/fc54b0233e186232f6d724fa89d1cf7c1f45c688
 
-| Auditors   | Types | Commits              | Reports                                                  |
-| :--------- | :---- | :------------------- | :------------------------------------------------------- |
-| Quantstamp | Firm  | [`fc54b02`][fc54b02] | [2019-11-25](./lockup/v1-legacy/20191125_quantstamp.pdf) |
+| Auditors   | Types | Commits              | Reports                                               |
+| :--------- | :---- | :------------------- | :---------------------------------------------------- |
+| Quantstamp | Firm  | [`fc54b02`][fc54b02] | [2019-11-25](./lockup/legacy/20191125_quantstamp.pdf) |
```
Comment (smol-ninja) [inline README.md:74]: it should be `./legacy/20191125_quantstamp.pdf`.

File: README.md (hunk 1 above)
Comment (andreivladbrg) [reply README.md:50]: similar to `package.json` file, what about: `v1.1.x`

File: README.md (hunk 2 above)
Comment (andreivladbrg) [reply README.md:74]: correct

**Review by smol-ninja — APPROVED** [review]

**Pull request merged**